packages/react/src/context/languageContext.tsx

```diff
@@ -9,7 +9,7 @@ export type LanguageOption = {
 };
 export type LanguageContextType = {
```

Comment:
It seems like this isn't exported in to the build? But this type is mentioned in `languages` prop for NavigationLanguageSelector.

Reply:
do you mean `type LanguageOption` or `LanguageContextType`? `LanguageOption`is used for prop `languages` and `LanguageContextType` for `LanguageContext`

Reply:
Ah sorry, I meant `LanguageOption`

Reply:
Maybe we should export `type LanguageOption` in `packages/react/src/components/header/index.ts` so it included in the build? So our TypeScript users could use it.